### PR TITLE
Getting Started with Burp Suite

### LINKED ISSUE #5756: Getting started with Burpsuite
### Topic Suggestion
Getting Started with Burpsuite

### Writing sample(s): 
https://www.section.io/engineering-education/understanding-cross-site-scripting-attacks/

#### Proposed title of article
Getting Started with Burpsuite

#### Proposed article introduction
This article seeks to introduce the reader to one of the most popular web application hacking tool.

[Security] Getting started with Burpsuite.

#### Key takeaways 

Readers will:
1. Be able to test for common web application vulnerabilities
2. Be able to intercept network traffic.
3. Be able to perform application security testing.

#### Article quality

This article will taking a practical approach in introducing the tool to the user.

#### References
N/A

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by shosenwales**
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #5756

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by Kimosop-Wakanda** [comment]
@ahmadmardeni1 
I would like to review this article
cc: @hectorkambow

**Comment by hectorkambow** [comment]
> @ahmadmardeni1 I would like to review this article cc: @hectorkambow

@Kimosop-Wakanda Have you submitted a "tryout" yet? That is before a test review - submitting a pull request that requires little to no revisions?

Once the tryout PR is approved then you can move on to a test review.

**Comment by Kimosop-Wakanda** [comment]
Okay @hectorkambow 
I'll ping you on the next

**Comment by ahmadmardeni1** [comment]
I believe you have submitted [an article](https://github.com/section-engineering-education/engineering-education/pull/5864) recently. This can be your tryout. @Kimosop-Wakanda

**Comment by Kimosop-Wakanda** [comment]
Okay @ahmadmardeni1 
So can I proceed with this

**Comment by ahmadmardeni1** [comment]
No. [This article](https://github.com/section-engineering-education/engineering-education/pull/5864) is your tryout. You have to wait until it is published and reviewed by our team. @Kimosop-Wakanda

**Comment by ahmadmardeni1** [comment]
Plagiarism check done ✅

File: content/articles/getting-started-with-burpsuite/index.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+Burpsuite is available as a free community edition and a professional edition which costs about $3999 a year.
+
+In this article, we'll be going through the basic usage of burp suite.
+
+**Disclaimer:** This article is for educational purposes only.
+
+### Prerequisites
+
+- Linux or Windows OS
+- Mozilla Firefox Browser
```
Comment (ahmadmardeni1) [inline content/articles/getting-started-with-burpsuite/index.md:27]: When you want to add the "Prerequisites," section, always add something like "To follow along with this tutorial, you are required to have the following:"

Delete the entire section. The user can use any browser.

File: content/articles/getting-started-with-burpsuite/index.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+### Burpsuite Configuration
```
Comment (ahmadmardeni1) [inline content/articles/getting-started-with-burpsuite/index.md:29]: Lowercase "Configuration".

File: content/articles/getting-started-with-burpsuite/index.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+![Start burp](/engineering-education/getting-started-with-burpsuite/start-burp.png)
+
+Burpsuite has been launched successfully.
+
+![Burpsuite download](/engineering-education/getting-started-with-burpsuite/burp.png)
+
+Now, we need to set up the burp suite proxy. The burp suite proxy allows us to intercept a web request while it is being processed. We're simply instructing our browser to use burp suite as a proxy, allowing us to intercept and alter requests while surfing our website.
+
+Firstly, we need to install a firefox extension called [Foxy Proxy](https://addons.mozilla.org/en-US/firefox/addon/foxyproxy-standard/). Click `Add to Firefox` to install the extension.
```
Comment (ahmadmardeni1) [inline content/articles/getting-started-with-burpsuite/index.md:53]: It is unnecessary to use Firefox, the reader can use any browser. Edit this, please.

File: content/articles/getting-started-with-burpsuite/index.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+After the installation, we'll get a little fox icon, by the address bar of our browser. Click on the icon, click on `options`, then click on `Add`
```
Comment (ahmadmardeni1) [inline content/articles/getting-started-with-burpsuite/index.md:55]: Add a dot at the end of the sentence. Please apply this to the entire article.

**Review by ahmadmardeni1 — COMMENTED** [review]
Also, please change the images' alt text. All of them are "Burpsuite download".

File: content/articles/getting-started-with-burpsuite/index.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
+Burpsuite is one of the most popular web application security tools. Burp Suite is an intercepting proxy tool that allows us to intercept, analyze and modify requests coming from our browsers before they are sent to the remote server. With burp suite we can intercept HTTP messages, modify the header and body of a message, and manually test for vulnerabilities.
+
+Burpsuite is one of those important tools found in the arsenal of web application penetration testers, web application security researchers, CTF players, as well as bug bounty hunters. 
+
+![Proxy](/engineering-education/getting-started-with-burpsuite/illustration.png)
```
[the remark this answers is not in the document]
Comment (shosenwales) [reply content/articles/getting-started-with-burpsuite/index.md:5]: Yes i did. Designed with canva

**Comment by shosenwales** [comment]
@ahmadmardeni1 thanks for the review. I made some changes, kindly go through.

**Comment by ahmadmardeni1** [comment]
You forgot to upload the hero image. @shosenwales

**Comment by shosenwales** [comment]
@ahmadmardeni1 added hero image

**Comment by ahmadmardeni1** [comment]
Great article. @shosenwales Ready for a final review.

**Comment by WanjaMIKE** [comment]
Nice work guys @shosenwales @ahmadmardeni1

**Comment by WanjaMIKE** [comment]
Merged #5843 into master

**Pull request merged**